external-service/Chart.yaml

```
@@ -1,4 +1,4 @@
 apiVersion: v1
 name: external-service
 description: Easily manage external services
-version: 0.1.1
+version: 0.1.2
```

Is a feature not a bugfix.
```suggestion
version: 0.2.0
```